viz.yaml

```diff
@@ -30,7 +30,7 @@ info:
       version: 2.2-6
     sf:
       repo: CRAN
-      version: 0.4.3
+      version: 0.5-4
```

Comment:
should we get `sp` in here too?

Reply:
for this PR, I don't use `sp`, but I think we do need it for #28